pontoon/base/models.py

```diff
@@ -1082,6 +1082,7 @@ class Repository(models.Model):
         choices=TYPE_CHOICES
     )
     url = models.CharField("URL", max_length=2000)
+    branch = models.CharField("Branch", default='master', max_length=2000)
```

Comment:
I tried a textField just now, but it created a multi line input box. Probably not what we want. I think 2000 characters is probably enough for a branch name ;) Is there another advantage to TextField?